app/views/audit.scala.html

```diff
@@ -772,6 +840,37 @@ <h1 class="text-center bold">Surface Problem</h1>
             svl.main = new Main(mainParam);
 
         }
+
+        @if(user){
+        //Survey Submission
+        $("#surveyForm").submit(function submitSurvey(e) {
+            var formData = $('#surveyForm').serializeArray();
+            //alert(formData);
+
+            var surveyUrl = '/survey';
+            var async = true;
+            //submit data from survey
+            $.ajax({
+                async: async,
+                contentType: 'application/json; charset=utf-8',
+                url: surveyUrl,
+                type: 'post',
+                dataType: 'json',
+                data: JSON.stringify({
+                    "answered_questions": formData,
+                    "is_admin":@user.get.roles.getOrElse(Seq("")).contains("Admin")
```

Comment:
What is this for?

Reply:
It turns out I cant have a json format with just a list of items (the "answered _questions" field is a list of question_id and value pairs). I added the is_admin field to prevent the errors that arose because of this. It is pretty much extra baggage.

Reply:
You can. Check out 1015 branch. I had a similar thing of sending a json with a list of items.

Reply:
Sorry but you need to be a little more specific than this. Also, to be more specific myself, I was facing issues with validating the form json object using the SurveySubmission format. The input to the associated case class SurveySubmission was a list of objects of another case class SurveySingleSubmission. This was causing some errors. I wasnt directly using validate([List[SurveySingleSubmission]]) because I realized we might want to send some additional information along with the user's survey responses.

Reply:
See IRRDataRequestFormat.scala. It deals with a json request with just a list of items.

Reply:
Okay, see if it works for you with the example file that I gave. If not, then leave it.

Reply:
I'd like to leave the way it is for now.